src/data/enum/Params.js

```diff
@@ -1,6 +1,4 @@
-const Params = {
+export default {
```

Comment:
We had all enums like this in vue skeleton but after a comment of Narie we changed it to the const with the seperate export default. It had something to do with webstorm. I am not a big fan of a direct export default. I almost always use the const with the separate export default. So when you read the code you understand what is exported. In this case you have to look at the filename but doesn't always work because you can have multiple exports in one file.

Reply:
Yeah, webstorm has/d better autocomplete / autoimport if the const has the same name as the file/import name.

Reply:
Fair point on the autocompletion, I did not consider this.